assets/base.css

```diff
@@ -267,13 +267,14 @@ body,
 
 .background-secondary {
   background-color: rgba(var(--color-foreground), 0.04);
-  padding: 4rem 0 5rem;
+  padding-top: 4rem;
+  padding-bottom: 5rem;
 }
 
 @media screen and (min-width: 750px) {
   .background-secondary {
-    padding: calc(6rem + var(--page-width-margin)) 0
-      calc(5rem + var(--page-width-margin));
+    padding-top: calc(6rem + var(--page-width-margin));
+    padding-bottom: calc(5rem + var(--page-width-margin));
```

Comment:
I feel like the padding used for background-secondary sections should scale a bit along with section spacing. https://screenshot.click/29-58-ddtp3-ewv3w.png 6/5rem seems out of place to me with a small section spacing value set. cc: @melissaperreault

Reply:
Good question! I'll think more about i!